draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+::
+
+  ZIP: Unassigned
+  Title: Block Reward Allocation for Non-Direct Development Funding
+  Owners: Kris Nuttycombe <kris@nutty.land>
+          Jason McGee <aquietinvestor@gmail.com>
+  Original-Authors: Skylar Saveland <skylar@free2z.com>
+  Credits: Daira-Emma Hopwood
+           Jack Grigg
+           @Peacemonger (Zcash Forum)
+  Status: Draft
+  Category: Consensus
+  Created: 2024-07-03
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/866>
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only
+when, they appear in all capitals.
+
+Abstract
+========
+
+This ZIP proposes several options for the allocation of a percentage of the
+Zcash block subsidy, post-November 2024 halving, to an in-protocol "lockbox."
+The "lockbox" will be a separate pool of issued funds tracked by the protocol,
+as described in ZIP <TBD>: Lockbox Funding Streams
+[#draft-nuttycom-lockbox-streams]_. No disbursement mechanism is currently defined
+for this "lockbox"; the Zcash community will need to decide upon and specify a
+suitable decentralized mechanism for permitting withdrawals from this lockbox
+in a future ZIP in order to make these funds available for funding grants to
+ecosystem participants.
+
+The proposed lockbox addresses significant issues observed with ZIP 1014
+[#zip-1014]_, such as regulatory risks, inefficiencies in funding organizations
```

Comment:
```suggestion
[#zip-1014]_, such as regulatory risks, inefficiencies due to funding organizations
```
To avoid a potential misreading of "inefficiencies" as applying to "funding organisations" rather than action of funding organisations instead of finding projects.